numpy/tests/test_ctypeslib.py

```diff
@@ -37,13 +38,15 @@
                     reason="Known to fail on cygwin")
 class TestLoadLibrary:
     def test_basic(self):
-        try:
```

Comment:
Note that the `try` / `except` clause is redundant due to the pytest fixture:

https://github.com/numpy/numpy/blob/a9edc0433e2efff6695c2d22a1cd0cad325556ae/numpy/tests/test_ctypeslib.py#L35-L39